src/cloud-api-adaptor/test/e2e/README.md

```diff
@@ -48,6 +48,26 @@ To leave the cluster untouched by the execution finish you should export `TEST_T
 
 To use existing cluster which have already installed Cloud API Adaptor, you should export `TEST_INSTALL_CAA=no`.
 
+## Attestation and KBS specific
+
+We need artificats from trustee when do attestation test, to prepare trustee, do as following. 
+
+```
+pushd ${cloud-api-adaptor}/test/e2e
+git clone https://github.com/confidential-containers/trustee.git
+pushd trustee
+git checkout $(../../../hack/yq-shim.sh '.git.kbs.reference' ../../../versions.yaml)
+pushd kbs
+make CLI_FEATURES=sample_only cli
+popd
+popd
+popd
+```
+
+To deploy the KBS service and test attestation related cases, set `DEPLOY_KBS=yes`.
```

Comment:
I was having issues with the kbs deployment not working and discovered that if I guess run `DEPLOY_KBS=yes` before `make test-e2e` then it isn't set when we check it in the code, so maybe we should explicitly state `export DEPLOY_KBS=yes` here?

Reply:
Yes, agreed